renconstruct/renconstruct.py

```diff
@@ -422,13 +422,10 @@ def cli(project, output, config, debug):
                 logger.info(
                     "Building {} packages".format(", ".join(platforms_to_build))
                 )
-            cmd = "renutil {} launch {} -h distribute \
-            {} --destination {}".format(
-                registry_cmd,
-                config["renutil"]["version"],
-                shlex.quote(config["project"]),
-                shlex.quote(config["output"]),
-            )
+            cmd = f'renutil {registry_cmd} launch "{config["renutil"]["version"]}" -h \
```

Comment:
The `shlex.quote` thing should achieve the same thing as your solution, but cross-platform. Have you tested the latest revision which includes these changes to see whether they work?
Also, to support older Python versions, I'v so far abstained from using f-strings since they were introduced in a later Python version than 3.7, which we still support.

Reply:
I didn't know f-string were so recent.

I tested the `shlex.quote` thing but it generated single quotes, that didn't work on my Windows.

Reply:
Reverted the f-string back to string formatting.

Reply:
Yep, thanks for the change. Could you also make the same change for line 402/403? Those also have shlex calls in them, so should be updated if it doesn't work. Once that's done I can merge this in.

Reply:
Added double quotes in line 402/403.